dino.icu.yaml

```diff
@@ -426,3 +426,8 @@ node:
 - ttl: 600
   type: CNAME
   value: tgh75972.github.io.
+
+gyat: 
```

Comment:
Hey @4H4CayJ,

At Hack Club we want to do our best to be as respectful as possible! Please edit your PR with a different / more appropriate subdomain.

Reply:
ok i changed it to terror .